src/runtime_src/core/edge/user/device_linux.h

```diff
@@ -91,6 +91,9 @@ class device_linux : public shim<device_edge>
   virtual void
   wait_ip_interrupt(xclInterruptNotifyHandle);
 
+  virtual std::cv_status
+  wait_ip_interrupt(xclInterruptNotifyHandle, int32_t timeout);
```

Comment:
```suggestion
  std::cv_status
  wait_ip_interrupt(xclInterruptNotifyHandle, int32_t timeout) override;
```
Feel free to change all other overrides in this file while you have it open for edit :-)

Reply:
Hi Soren, I made changes as per your suggestion. Can you please review once.